testapps/testapp/src/main/res/raw/msal_config_default.json

```diff
@@ -2,8 +2,9 @@
   "client_id" : "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0",
   "authorization_user_agent" : "DEFAULT",
   "redirect_uri" : "msauth://com.msft.identity.client.sample.local/1wIqXSqBj7w%2Bh11ZifsnqwgyKrY%3D",
-  "handle_null_taskaffinity": true,
+  "handle_null_taskaffinity": false,
```

Comment:
Do we have new docs covering this?

Reply:
I have to write these docs specifically for Samsung... not sure that we want to document this in public yet.